reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -979,10 +979,16 @@
 
 	/**
 	 * Create a {@link Flux} that emits the items contained in the provided {@link Iterable}.
-	 * A new iterator will be created for each subscriber.
+	 * The {@link Iterable#iterator()} method will be invoked at least once and at most twice
+	 * for each subscriber (see onDiscard information below).
 	 * <p>
 	 * <img class="marble" src="doc-files/marbles/fromIterable.svg" alt="">
 	 *
+	 * @reactor.discard Upon cancellation, this operator attempts to discard remainder of the
```

Comment:
I'd put this attribute after the double `iterator()` clarification, otherwise it reads as a detail of `discard` and will be ignored

Reply:
actually I see that Intellij still doesn't show all the content of these custom tags. Maybe we need to rework all that part of the javadoc to mention the discard caveats in the body and just use the tags as markers or something 🤔

Reply:
in the meantime, the tag is only rendered out of the body of the javadoc when generating HTML

Reply:
@bsideup I have reworked the javadoc a bit, introducing a bit of duplication as an example of how info could be split between tag and javadoc body (so that IDE still convey important information if they don't display the tags)